roles/telemetry_graphing/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: "[Test] Enable Graphing"
+  ansible.builtin.shell:
+    cmd: |
+      oc patch oscp/controlplane --type='json' -p '[{"op": "replace", "path": "/spec/telemetry/template/metricStorage/dashboardsEnabled", "value":true}]'
+  register: oscp_change
+  changed_when: false
+
```

Comment:
This should be in ci/run_graphing_test, as it is set-up for the test, rather than the test itself.

The change should also be reverted after (also not in the role)

A further note, you should wait for the change to be applied, in case it takes a while to resolve.
This should not be marked with "TEST" as it is setup.